yesod-auth/Yesod/Auth/Email.hs

```diff
@@ -290,6 +291,17 @@ class ( YesodAuth site
     normalizeEmailAddress :: site -> Text -> Text
     normalizeEmailAddress _ = TS.toLower
 
+    -- | Handler called to render the login page.
+    -- The default works fine, but you may want to override it in
+    -- order to have a different DOM.
+    --
+    -- Default: 'defaultEmailLoginHandler'.
+    --
+    -- @since 1.4.11
```

Comment:
This has to be `1.4.17` since the function `emailLoginHandler` will be exported as part of the typeclass from it's next version.